src/components/Overlay/OverlayTop/index.tsx

```diff
@@ -45,16 +45,16 @@ const StyledWrapper = styled.div`
   }
 `;
 
-const StyledNewsSection = styled.section`
+/* const StyledNewsSection = styled.section`
```

Comment:
Should we keep dead code around?

Reply:
In general I'd say we shouldn't.

In this case I've left it in there because the section will most likely come back and it seems like less work to comment it back in than getting it from the git history.

Reply:
As you like